ingestion-beam/pom.xml

```diff
@@ -60,6 +60,11 @@
             <artifactId>beam-sdks-java-io-google-cloud-platform</artifactId>
             <version>${beam.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-kms</artifactId>
+            <version>0.81.0-beta</version>
```

Comment:
Is it possible to keep this in step with `${google-cloud.version}`? If not, we should add a comment to explain why.

Reply:
The latest [stable version is 1.38.0](https://cloud.google.com/kms/docs/reference/libraries#installing_the_client_library), and the `google-cloud.version` is pinned to the beam dependency's version of 1.61.0. I ran into authentication issues ([in a similar fashion to this](https://github.com/GoogleCloudPlatform/java-docs-samples/issues/1377)) that were resolved by changing to the beta releases which are more up to date.